modules/libinput.yml

```diff
@@ -0,0 +1,49 @@
+name: libinput
+buildsystem: meson
+config-opts:
+  - --libexec=lib
+  - -Dlibwacom=false
+  - -Ddebug-gui=false
+  - -Dtests=false
+  - -Ddocumentation=false
+  - -Dzshcompletiondir=no
+sources:
+  - type: archive
+    url: https://gitlab.freedesktop.org/libinput/libinput/-/archive/1.23.0/libinput-1.23.0.tar.gz
+    sha256: 7a7c90fbc59f1f65c781a51fa634e4f79e460bf6e16ad68afbe7965d25d09738
+    x-checker-data:
+      type: anitya
+      project-id: 5781
+      stable-only: true
+      url-template: https://gitlab.freedesktop.org/libinput/libinput/-/archive/$version/libinput-$version.tar.gz
+cleanup:
+  - /bin
+  - /etc
+  - /lib/libinput
+  - /lib/udev
+  - /share
+  - /include
+  - /lib/pkgconfig
+modules:
+  - name: mtdev
+    buildsystem: autotools
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://bitmath.org/code/mtdev/mtdev-1.1.6.tar.bz2
+        sha256: 15d7b28da8ac71d8bc8c9287c2045fd174267bc740bec10cfda332dc1204e0e0
+        x-checker-data:
+          type: anitya
+          project-id: 8868
+          stable-only: true
+          url-template: https://bitmath.org/code/mtdev/mtdev-$version.tar.bz2
+      # Upstream uses an outdated version of the config.guess/config.sub script,
+      # so we patch it here to fix aarch64 builds up.
+      - type: patch
+        path: config-update-fix.patch
```

Comment:
you should instead just copy them and not use a patch. Like virtually any other package that need to do that.

Example:
https://github.com/flathub/org.gimp.GIMP/blob/master/org.gimp.GIMP.json#L300